pom.xml

```
@@ -547,9 +536,9 @@
             </build>
         </profile>
         <profile>
-            <id>jdk11</id>
+            <id>jdk8plus</id>
```

maybe this is better - `jdk8plus` makes me think jdk8 _and_ newer 
```suggestion
            <id>newerThanJdk8</id>
```